mesonbuild/compilers/mixins/emscripten.py

```diff
@@ -63,7 +63,8 @@ def get_options(self) -> coredata.MutableKeyedOptionDictType:
         opts[key] = options.UserIntegerOption(
             self.make_option_name(key),
             'Number of threads to use in web assembly, set to 0 to disable',
-            (0, None, 4))  # Default was picked at random
+            4,  # Default was picked at random
+            min_value=0)
```

Comment:
I think this may be a good case where it's worth explaining the change in the commit message. After thinking about this, I think that this works because the dataclass also fits a historic mess where the "value" member isn't consistently used by all option types, when we do stuff like overloading it with minimum/maximum values instead?